src/Wallabag/UserBundle/Resources/views/Authentication/form.html.twig

```diff
@@ -1,5 +1,5 @@
 {# Override `vendor/scheb/two-factor-bundle/Resources/views/Authentication/form.html.twig` #}
-{% extends "WallabagUserBundle::layout.html.twig" %}
+{% extends "@WallabagUser/layout.html.twig" %}
```

Comment:
I think it should be
```suggestion
{% extends "@FOSUser/layout.html.twig" %}
```
See https://github.com/wallabag/wallabag/pull/5833

Reply:
updated! thanks

Reply:
@j0k3r shouldn't `master` be merged first in `2.6.0`? so that the fix properly exists first in the branch, and then this PR be about migrating from `FOSUserBundle::layout.html.twig` to `@FOSUser/layout.html.twig`

Reply:
Yup, I'll release 2.5.1 soon and then merge master into 2.6.0

Reply:
good! just ping me when done so I update this :)

Reply:
Updated, you can rebase.